crates/net/discv4/src/lib.rs

```diff
@@ -283,6 +288,33 @@ impl Discv4 {
         self.lookup_node(Some(node_id)).await
     }
 
+    /// Performs a random lookup for node records.
+    pub async fn lookup_random(&self) -> Result<Vec<NodeRecord>, Discv4Error> {
+        self.new_random_lookup().await
+    }
+
+    /// Creates a new random lookup for nodes.
+    ///
+    /// This method generates a random target using cryptographic random number generation
+    /// and then initiates a node lookup based on this target.
+    ///
+    /// # Steps
+    /// 1. Generates a random `FixedBytes<64>` as the target.
+    /// 2. Calls `lookup_node` with the generated target.
+    ///
+    /// # Returns
+    /// A `Result` which is `Ok` with a `Vec<NodeRecord>` if the lookup is successful, or
+    /// `Err(Discv4Error)` in case of an error during the process.
+    ///
+    /// # Errors
+    /// Returns `Discv4Error` if the lookup process fails at any point.
```

Comment:
this can be simplified by simply referencing that this calls [Self::lookup_node] with a new random PeerId